src/component/menu/ApiDetails/ApiDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import useUiConfig from '../../../hooks/api/getters/useUiConfig/useUiConfig';
+import { ReactElement } from 'react';
+import ConditionallyRender from '../../common/ConditionallyRender';
+import { IVersionInfo } from '../../../interfaces/uiConfig';
+
```

Comment:
If this component is only used in the Footer, I think we can actually move it to the Footer directory and have it live under there.